### ISSUE TITLE
Wired loss and reconstruction results when training the autoencoder

### ISSUE THREAD

**Issue by OwalnutO** [CompVis/latent-diffusion#309]
I'm trying to train the AE on my own dataset (~10w) with the default config file autoencoder_kl_32x32x4.yaml. I only decrease the learning rate from 4.5e-6 to 1e-6 since I use a smaller batchsize. However, the training losses are wired and the reconstructiuon results are unsatisfactory. Could anyone give some suggestions? Thanks in advance!
![3](https://github.com/CompVis/l
![2](https://github.com/CompVis/latent-diffusion/assets/24652623/5292bb36-8174-49ea-b8c7-83ee85fabb0d)
![2](https://github.com/CompVis/latent-diffusion/assets/24652623/39cf9427-68da-4640-bc52-636b61aab8a1)
![1](https://github.com/CompVis/latent-diffusion/assets/24652623/20f098d5-2e8a-4c84-a193-e1e9594cad10)
Recon results![4](https://github.com/CompVis/latent-diffusion/assets/24652623/62c525ad-a042-47da-be65-3eaf199aee8b)    GT image![5](https://github.com/CompVis/latent-diffusion/assets/24652623/dcbedddf-c85b-4f77-bf52-5452a1eedd73)

**Comment by illrayy** [comment]
I think it might be that kl_weight is too small, you can try increasing kl_weight in the config

**Comment by houqingying** [comment]
@OwalnutO Hi, I encounter the same problem, have you solved it by increasing kl_weight? thank you!

**Comment by GuHuangAI** [comment]
You may find answers in here #187 .
By the way, why does the reconstruction have a different size to the GT image?

**Comment by Echo-jyt** [comment]
Hello, can you share how to train AE on your own data set?

**Comment by xiaoerjason** [comment]
I also encountered this problem, and interestingly, the Autoencoder KL reconstruction effect was good in my previous code. But I encountered this issue while retraining on other devices.
